Gemfile

```diff
@@ -0,0 +1,48 @@
+source 'https://rubygems.org'
+ruby '2.2.3'
+
+gem 'rails', '4.2.4'
+gem 'haml-rails'
```

Comment:
You should be able to remove haml-rails bootstrap-sass autoprefixer-rails attr_encrypted sinatra sidekiq sidekiq-cron slim as they are now included in the framework dependancies